src/Output/Stdout.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Codelicia\Xulieta\Output;
+
+use Codelicia\Xulieta\ValueObject\Violation;
+use Symfony\Component\Console\Output\OutputInterface;
+use const PHP_EOL;
+
+final class Stdout implements OutputFormatter
+{
+    private OutputInterface $output;
+
+    public function __construct(OutputInterface $output)
+    {
+        $this->output = $output;
+    }
+
+    public function addViolation(Violation $violation) : void
+    {
+        $this->writeln('<error>Wrong code on file: ' . $violation->code()->file() . '</error>');
+        $this->writeln($violation->message() . PHP_EOL);
+        $this->writeln($violation->code()->code());
```

Comment:
Would be nice to show the file line here?

Reply:
We will be working a new `stdout` with syntax highlight and a proper way to point error in specific line. It would be nice to have an issue for that.

Reply:
nice. I'll create an issue for that.